### DISCUSSION TITLE
what does the HA entities

### DISCUSSION THREAD

**Discussion by 1mfaasj** [krahabb/meross_lan#444] [Q&A]
Hi can someone explain what the home assistant entities does?
I enabled meross lan to use with my own mqtt broker and some of the entities are not known to me.
for instance: buzzerEnable, Dnd? 
but also signalclose and signalopen? 

thanks!

**Comment by krahabb** [comment]
Hello @1mfaasj,
You are likely referring to a garage door device and those 'strange' names are more or less the same labels used in Meross protocol.
As far as I know:
`Dnd`: is a form of 'presence light' (do not disturb) available on almost every Meross device
`buzzerEnable`: is typically found in garage door devices and I guess it is a feature to enable/disable a sort of warning/alarm sound in the device
`signalClose/Open`: are usually the timeouts for the opening/closing run of the garage door even if these values might have been renamed lately since new garage doors have `door(Open/Close)Duration` which looks more meaningful for that. If that is the case I might guess these `signalClose/Open` are more likely related to the duration of the pulse sent by the garage door device to the garage controller to actually trigger the movement
